src/Microsoft.AspNetCore.Mvc.Razor.Host/ViewComponentTagHelperPass.cs

```diff
@@ -11,21 +11,21 @@
 
 namespace Microsoft.AspNetCore.Mvc.Razor.Host
 {
-    public class ViewComponentTagHelperPass : IRazorIRPass
+    public class ViewComponentTagHelperPass : IRazorIROptimizationPass
     {
         public RazorEngine Engine { get; set; }
 
-        public int Order => RazorIRPass.LoweringOrder;
+        public int Order => RazorIRPass.DefaultFeatureOrder;
```

Comment:
Q: Why not have everything based around 0 and have the `DefaultFeatureOrder` be 0?